### PR TITLE
Chore: Refactor Multiple Functions 🧷 

### PR DESCRIPTION
**Pull request by yezz123**
##### Checklist

- [x] `npm install && npm test` passes
- [ ] tests are included 
- [ ] documentation is changed or added
- [x] commit message follows [commit guidelines](https://github.com/googleapis/release-please#how-should-i-write-my-commits)

##### Description of change

This is just a refactoring for multiple functions in the Python codebase, just to let its many multiple changes in the PEP rules and guidelines. Any Issue I'm here to discuss 🚀

### CODE REVIEW

File: gyp/tools/pretty_gyp.py [hunk 1]
```diff
@@ -126,14 +126,13 @@ def prettyprint_input(lines):
...
             if len(line) > 0:
                 (brace_diff, after) = count_braces(line)
-                if brace_diff != 0:
-                    if after:
-                        print(" " * (basic_offset * indent) + line)
-                        indent += brace_diff
-                    else:
-                        indent += brace_diff
-                        print(" " * (basic_offset * indent) + line)
+                if brace_diff == 0:
```
Comment (cclauss) [inline gyp/tools/pretty_gyp.py:129]: I do not see an advantage to risking this change.

Unless of course, we have pytests that cover all cases.

**Review by cclauss — CHANGES_REQUESTED** [review]

**Review by richardlau — CHANGES_REQUESTED** [review]
These need to be made in the upstream https://github.com/nodejs/gyp-nex repository.

**Comment by yezz123** [comment]
> These need to be made in the upstream https://github.com/nodejs/gyp-next repository.

If this merged here I can open another PR in the Upstream Repository.

**Comment by rvagg** [comment]
@yezz123 sorry, no, we pull from that repo to here - we even have a script to do it in the project root. But maybe now that you've made the changes here you can copy the files over the top of a clone of gyp-next, we may be in sync with the current version already so it might not be too painful. Thanks for the contribution though!

**Pull request closed without merging**